### PR TITLE
SVLS-4605 - Improve agent and tracer logs in AAS Linux

### PR DESCRIPTION
**Pull request by TophrC-dd**
### What this does 
This PR updates the datadog wrapper to set the trace logging directory which will allow a consistent logging experience when combined with the configuration files also provided in this script. It will keep agent and dogstatsD logs in a single place based on the version of the wrapper.

Also some changes were made to process_manager to support the addition arguments of the Dogstatd listener. Process manager only previously passed on additional argument. Now it passes two more arguments, the config location flag and the path. 

I tested the wrapper in a docker container to ensure we see the expected behavior from the the script. Along with running the dockerfile end to end. This ensured the package generated the zip file with the configuration file as intended.

### CODE REVIEW

File: datadog.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## disable sending the host metadata payload
+enable_metadata_collection: false
+apm_config:
+  enabled: true
```
Comment (jcstorms1) [inline datadog.yaml:4]: I think we can eliminate this part actually. It is set by the customer as env vars. Do you have time to test it?

File: dogstatsd.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## disable sending the host metadata payload
+enable_metadata_collection: false
+use_dogstatsd: true
```
Comment (jcstorms1) [inline dogstatsd.yaml:3]: Same thing here as I mentioned above.

**Review by jcstorms1 — CHANGES_REQUESTED** [review]

**Review by jcstorms1 — APPROVED** [review]

File: datadog_wrapper [hunk 3]
```diff
@@ -61,11 +61,15 @@ setEnvVars() {
     fi
 
     if [ -z "${DD_BINARIES_URL}" ]; then
-        DD_BINARIES_URL="https://github.com/DataDog/datadog-aas-linux/releases/download"
+        DD_BINARIES_URL="https://github.com/TophrC-dd/datadog-aas-linux/releases/download"
```
Comment (duncanpharvey) [inline datadog_wrapper:64]: Change back to DataDog

**Review by duncanpharvey — CHANGES_REQUESTED** [review]
LGTM with one minor fix!

File: datadog_wrapper (hunk 3 above)
Comment (TophrC-dd) [reply datadog_wrapper]: Done!

**Review by duncanpharvey — APPROVED** [review]

**Pull request merged**